Update Account Setup Docs Page (#44)
datum-cloud/docs: content/en/docs/glossary.md

A specification that defines the configuration for compute instances within a
workload, including machine type, image, storage, and network attachments.
Instance templates enable consistent and repeatable deployments across different
locations.

## IP Address Management (IPAM)

The automated allocation and management of IP addresses within Datum Cloud
networks. IPAM ensures efficient use of address space and prevents conflicts
across distributed workload deployments.

## Location

A geographical and cloud provider context where Datum workloads can be deployed.
Locations define the available infrastructure zones and provide the foundation
for workload placement decisions based on latency, compliance, or performance
requirements.

## Network

A virtual private cloud (VPC) network that defines how workloads communicate
within Datum Cloud. Networks provide isolated networking environments with
configurable subnets, routing, and security policies.

## Network Binding

A resource that defines the relationship between workloads and networks,
specifying how instances should be attached to specific network segments and
what connectivity policies apply.

## Network Context

> A logical grouping that provides network configuration and policy context for
> workload deployments. Network contexts help organize and manage networking
> resources across different environments or applications.

Reviewer: ```suggestion
A logical partition of a Network that helps organize and manage networking
resources such as Subnets across different Locations. A functioning Network will have one or more Network Contexts.
```

## Network Function Virtualization (NFV)

The virtualization of network services that traditionally ran on dedicated
hardware. Datum Cloud supports deployment and lifecycle management of both
commercial and open source NFV technologies as software-based workloads.

## Network Policy

> Security rules that control traffic flow between workloads, networks, and
> external resources. Network policies provide fine-grained access control and
> segmentation within Datum Cloud environments.

Reviewer: ```suggestion
Security rules that control traffic flow between endpoints on a network, and
external resources. Network policies provide fine-grained access control and
segmentation within Datum Cloud environments.
```

## Open Network Cloud

Datum's vision for a network infrastructure platform that can run anywhere - on
managed global infrastructure or federated with customer-controlled locations -
while being built on open source technologies under the AGPL v3 license.

## Placement Rules

Configuration that determines where workload instances should be deployed across
available locations and providers. Placement rules consider factors like latency
requirements, compliance needs, resource availability, and cost optimization.

## Provider Operator

A software component that manages the lifecycle of infrastructure resources on
specific cloud providers (e.g., Google Cloud, AWS, Azure). Provider operators
translate Datum Cloud resource definitions into provider-specific actions like
creating VMs or configuring networks.

## Reconciliation

The continuous process of ensuring that the actual state of infrastructure
resources matches the desired state defined in API resource specifications.
Reconciliation automatically handles failures, scaling, and configuration drift.

## Scaling Behavior

Configuration that defines how workloads should automatically scale in response

> to demand, resource utilization, or other triggers. Scaling behavior includes
> policies for both horizontal scaling (adding instances).

Reviewer: ```suggestion
to demand, resource utilization, or other triggers. Scaling behavior includes
policies for minimum and maximum replicas, and horizontal scaling expectations.
```

## Service Chaining

The ability to route traffic through a sequence of network services or
functions, enabling complex traffic processing workflows. Service chaining
allows for advanced traffic management, security filtering, and protocol
transformations.

## Subnet

A network segment within a larger Network that provides IP address allocation
and routing boundaries. Subnets can provide the basic connectivity fabric for
workload instances.

## SubnetClaim

A request for subnet resources that automatically provisions the necessary

> network infrastructure. SubnetClaims provide a declarative way to request
> network capacity without managing low-level networking details.

Reviewer: ```suggestion
network infrastructure. SubnetClaims provide a declarative way to request
network addresses for use on a Network while allowing for IPAM policies to decide what addresses should be issued.
```

## Virtual Machine Workload

A type of workload that runs applications on traditional virtual machines rather
than containers. VM workloads provide full operating system isolation and are
suitable for legacy applications or specific compliance requirements.

## Volume Mount

Storage attachment configuration for workload instances, defining how storage

> volumes should be connected to running compute instances. Volume mounts enable
> stateful workloads with data persistence.

Reviewer: ```suggestion
volumes should be connected to running compute instances. Volume mounts enable
stateful workloads with data persistence, as well as injecting content from ConfigMaps or Secrets via a filesystem path.
```

## Workload

A provider-agnostic specification for managing groups of compute instances (VMs
or containers) including their configuration, placement, scaling, networking,
and storage requirements. Workloads are the primary unit of application
deployment in Datum Cloud.

## Workload Deployment

> The actual running instances and infrastructure created from a Workload
> specification. Workload deployments represent the realized state of applications
> across the distributed infrastructure.

Reviewer: ```suggestion
A partition of a Workload created as a result of placement rules. Each Workload Deployment is responsible for maintaining the lifecycle of Instances as defined by the placement rule's scale settings. A single Workload may have one or more Workload Deployments, with each being individually responsible for its set of instances.
```
